src/Symfony/Component/Notifier/Bridge/LineBot/Tests/LineBotTransportTest.php

```diff
@@ -47,6 +51,53 @@ public static function unsupportedMessagesProvider(): iterable
         yield [new DummyMessage()];
     }
 
+    public static function validMessageProvider(): iterable
```

Comment:
```suggestion
    public static function validMessageProvider(): array
```
or rework the data provider to use `yield`